netlink/netlink_linux.go

```diff
@@ -1213,6 +1214,7 @@ func AddToBridge(iface, master *net.Interface) error {
 }
 
 func randMacAddr() string {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Should it be seeded for each call, or once? (A similar discussion was in docker recently; https://github.com/docker/docker/pull/12214)